### ISSUE TITLE
Bug after multiple backups

### ISSUE THREAD

**Issue by jdbruxelles** [Microsoft/vscode#47259]
Issue Type: <b>Bug</b>

Hi VS Code Team and users,

I am currently developing a web application of about 6 MB, including 107 files (including compressed files).
Since 2017, when I code, I have several files (<5 files) open at the same time. Over the savings,  the VS code program has bugs, I notice it by the stopping update of the extension "Live server".
This forces me to restart the program each time.
Despite a reboot and reinstallation of the system, the problem persists.

I hope that you can help me with these informations.

VS Code version: Code 1.22.0 (6b4d53cdab8bcae1eaaa4934d93c077319b573db, 2018-04-05T17:27:42.363Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Celeron(R) CPU  N2930  @ 1.83GHz (4 x 1833)|
|Memory (System)|3.89GB (0.89GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
jshint|dba|0.10.18
vscode-eslint|dba|1.4.8
vscode-html-css|ecm|0.2.0
auto-rename-tag|for|0.

**Comment by roblourens** [comment]
> Over the savings, the VS code program has bugs, I notice it by the stopping update of the extension "Live server".

Sorry, I don't understand what the problem is. Can you explain a little more?

**Comment by jdbruxelles** [comment]
Excuse me, I'm not very good at English.
I'm going to explain. When I use VS code to write my project which contains `107 files` (including compressed files) and makes about `6 MB`, I'm having problems with the app performance. I don't know if it's caused by the low memory system (`0.89GB free/3.89GB`) or just the server extensions.
The problem is that: The most used files (no compressed) contains about 3000 lines each. When I code, I need to open 2 to 4 files.

I noticed that, if the server extension, `Live Server` isn't active, VS Code works very well, else, the VS Code performance decreases after many changes and backups, and It's so slow that even saving the file becomes slow and the localhost server stops working, hint, autocompletion, emmet and other functionalities stop working too. In this case, I can't continue and I'm forced to restart the app.

I have tried two localhost server extensions and I have reboot my computer too (for another reason) but the problems persist.

I hope that's more explained now.
